notebooks/samples/scikit-learn/custom_prediction/custom-prediction-routine-scikit-learn.ipynb

```diff
@@ -0,0 +1,1007 @@
+{
```

Comment:
<p>Not sure if this notebook should use latest TF 1.15 + Python 3.7?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/149/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>